src/EditorFeatures/Core.Wpf/Interactive/InteractiveSupportsFeatureService.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.CodeAnalysis.Shared;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+{
+    internal sealed class InteractiveSupportsFeatureService
+    {
+        [ExportWorkspaceService(typeof(ITextBufferSupportsFeatureService), WorkspaceKind.Interactive), Shared]
+        internal class InteractiveTextBufferSupportsFeatureService : ITextBufferSupportsFeatureService
```

Comment:
you can have 1 type with 2 interfaces with 2 ExportWorkspaceServices?

you can even share instance by exporting ExportWorkspaceServiceFactory with same service instance?

Reply:
No, you can't. Each export is a nested type itself.